my-profile-webapp/src/main/webapp/partials/countries.html

```diff
@@ -0,0 +1,251 @@
+<select id="countries" class="form-control" required ng-model="contact.addresses[0].country">
```

Comment:
We want to keep the same JSON object for emergency and local so we will keep it an array. Talk in the room says 1 address is cool for emergency.